src/test/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifierIntegrationTest.java

```diff
@@ -115,6 +136,11 @@ private void mockHttpWorker() {
         }
     }
 
+    private void mockGetChangeSets() {
+        List<ChangeLogSet> files = new AffectedFileBuilder().sampleFiles(run);
+        when(run.getChangeSets()).thenReturn(files);
+    }
+
```

Comment:
👻 whitespace or at least double empty line

Reply:
that was intentional because this is where mock section ends and test section starts - once this is merged I will provide more tests for other statuses